pymc3/__init__.py

```diff
@@ -34,3 +34,21 @@
     _log.setLevel(logging.INFO)
     handler = logging.StreamHandler()
     _log.addHandler(handler)
+
+def tqdm(*args, **kwargs):
+    try:
+        from tqdm import tqdm_notebook
+        return tqdm_notebook(*args, **kwargs)
+    except:
```

Comment:
I tried this via `jupyter console` and `jupyter qtconsole`.  There is no error, but no progress bar is displayed.  So `tqdm_notebook` doesn't error but doesn't display, meaning the try/except logic isn't enough here.  Maybe it makes more sense to close this PR until tqdm addresses this.